### DISCUSSION TITLE
求助贴

### DISCUSSION THREAD

**Discussion by Pai-pi** [share121/inter-knot#750] [其他]
请问绳网怎么进啊？我看到哔哩哔哩上的视频【🔥全网首发，我还原了绳网🔥-哔哩哔哩】 https://b23.tv/JdQVJUi
有个大佬做了个绳网，但是进不去，我没有邦布，也没有萝卜，请问有没有绳匠帮帮我[doge]

**Comment by wushanef** [comment]
你这不已经进了吗？

**Comment by QingMingqwq** [comment]
这个委托我接了...等等？这里不就是绳网嘛..

**Comment by aaa00073** [comment]
求求大佬们怎么评论啊，我刚进绳网什么也不懂，不会发帖也不会评论，有没有懂得教教我！
